app/filter-helpers.js

```diff
@@ -10,19 +10,60 @@ for (var group in formatMappingData) {
 
 var sectionFilter = function (allContent, pageSection) {
   console.log("Filtering " + allContent.length + " items by " + pageSection);
+  var filteredList = _filteredList(allContent, pageSection);
+
+  console.log("Filtered out " + filteredList.length);
+  console.log(filteredList);
+
+  return filteredList;
+}
+
+var rowsWithThreeColumns = function(allContent, pageSection) {
```

Comment:
We did, although I found it easier to come with a solution using javascript, and I realised I wouldn't know of how to only show a maximum of 6 elements, hence this solution.